.travis.yml

```diff
@@ -25,35 +24,50 @@ before_install:
 install: travis_retry composer update --optimize-autoloader --prefer-dist --prefer-stable --no-progress --no-interaction -vv $COMPOSER_FLAGS
 
 script:
-  - composer validate --strict
```

Comment:
You had asked that I add --strict when I added the validate command, any particular reason you're removing this here?

Reply:
No.

Reply:
The strict option is available again. Just of Copy mistake.